catalog/registry/catalog-registry-cswinputtransformer/pom.xml

```diff
@@ -91,6 +83,11 @@
             <artifactId>platform-parser-xml</artifactId>
```

Comment:
This is actually a duplicate, fix while here? Also select if scope should be test for one left.